test/parallel/test-timers-setunreftimeout.js

```diff
@@ -0,0 +1,58 @@
+// Flags: --expose-internals
+'use strict';
+
+const common = require('../common');
+
+const { strictEqual } = require('assert');
+const { setUnrefTimeout } = require('internal/timers');
+
+{
+  common.expectsError(
+    () => setUnrefTimeout(),
+    {
+      code: 'ERR_INVALID_CALLBACK',
+      type: TypeError,
+      message: 'Callback must be a function'
+    }
+  );
+}
+
+// Test this if correctly passes arguments to the callback
```

Comment:
`Test that setUnrefTimeout correctly passes arguments to the callback` ?

Reply:
Exactly.I fix it.